pkg/stdcopy/stdcopy.go

```diff
@@ -33,6 +33,9 @@ const (
 
 var bufPool = &sync.Pool{New: func() interface{} { return bytes.NewBuffer(nil) }}
 
+// Exported errors
+var ErrInvalidStdHeader = errors.New("Unrecognized input header")
```

Comment:
This should at least have a proper GoDoc to describe what it's used for. The error message also shouldn't be capitalised to follow Go conventions.

I'm wondering if it should implement a specific [errdef](https://github.com/moby/moby/blob/master/errdefs/defs.go), but I only glanced over, so would have to look at that 😅